src/pages/Login.tsx

```diff
@@ -0,0 +1,15 @@
+function Login() {
+  return (
+    <div className="register-container">
+      <h1>Login</h1>
+
+      <form>
+        <input className="reg-input" type="text" placeholder="Email" />
+        <input className="reg-input" type="text" placeholder="Password" />
+        <input className="submit-button" type="submit" value="Login" />
+      </form>
+    </div>
+  );
+}
+
+export default Login;
```

Comment:
Let's not use default export. use this syntax:
export const Login = () => {
....
}